src/main/java/ch/njol/skript/expressions/ExprAttacked.java

```diff
@@ -90,7 +97,10 @@ public boolean init(final Expression<?>[] vars, final int matchedPattern, final
 		final Entity[] one = (Entity[]) Array.newInstance(type.getType(), 1);
 		Entity entity;
 		if (e instanceof EntityEvent)
-			entity = ((EntityEvent) e).getEntity();
+			if (e instanceof ProjectileHitEvent && SUPPORT_PROJECTILE_HIT)
```

Comment:
minor nitpick but wouldn't it make more sense to check if project hit is supported before checking the class of `e`?